### ISSUE TITLE
How to deal with missing libraries and addons ?

### ISSUE THREAD

**Issue by gnouts** [flathub/tv.kodi.Kodi#111]
Hi,

New to Kodi, I installed the flatpak version 19. I started messing around with addon but sometimes they require some specific libraries.
For example, [Netflix addon from Castagnait](https://github.com/CastagnaIT/plugin.video.netflix/wiki/How-install-the-addon) requires `pycryptodomex`.

Right now, I'm unable to use this plugin. Not sure to whom it should fall under. I thought maybe you could either integrate it or redirect me to some relevant flatpak documentation.

**Comment by JeanOlivier** [comment]
I second this.

I did manage to install Netflix add-on in Kodi Leia, but now I can't get it to work no matter what I do.
The Netflix addon installs correctly, but when it launches it fails saying that "There's no module name 'Crypto'".

Just installing the cryptodomex python library in the flatpak should fix this. I don't know enough about flatpaks to be able to test this and submit a pull request though. 

Any help would be much appreciated!

**Comment by JeanOlivier** [comment]
I messed around a little and I found a solution that's appropriate for a personal install, although proper integration is the flatpak would be appreciated. The problem seems to be that if the host system doesn't use the same python version as that in the flatpak, the modules of the host system won't be available from inside the flatpak. In my case, the system was using python 3.9 whereas tv.kodi.Kodi uses 3.8.

For reference, you can enable Netflix support this way:
1. Launch a bash shell inside the flatpak: `flatpak run --command=bash tv.kodi.Kodi`
You should have a command line prepended with something like "[📦 tv.kodi.Kodi ~]$" now
2. On this command line, execute this to install pip and the proper libraries:
```
wget https://bootstrap.pypa.io/get-pip.py
python3.8 get-pip.py
python3.8 -m pip install --user Crypto pycryptodomex
```
3. Restart Kodi:
```
flatpak kill tv.kodi.Kodi
flatpak run tv.kodi.Kodi
```
4. Enjoy!

**Comment by wsnipex** [comment]
shoud be fixed in 19.1
